### PR TITLE
Issue 211

### PR DESCRIPTION
**Pull request by philippguertler**

### CODE REVIEW

File: core/src/main/java/com/gentics/mesh/core/verticle/node/BinaryFieldHandler.java [hunk 1]
```diff
@@ -396,26 +401,21 @@ public void handleTransformImage(RoutingContext rc, String uuid, String fieldNam
...
-		String sha512sum = hashFileupload(fileUpload);
 		checkUploadFolderExists(uploadFolder);
 		deletePotentialUpload(targetPath);
-		moveUploadIntoPlace(fileUpload, targetPath);
-		return sha512sum;
-		// log.error("Failed to handle upload file from {" + fileUpload.uploadedFileName() + "} / {" + targetPath + "}", error);
-		// throw error(INTERNAL_SERVER_ERROR, "node_error_upload_failed", error);
-
+		moveUploadIntoPlace(ac.get("sourceFile"), targetPath);
+		ac.put("sourceFile", targetPath);
```
Comment (Jotschi) [inline core/src/main/java/com/gentics/mesh/core/verticle/node/BinaryFieldHandler.java]: TODO: Document why you override the sourceFile.

File: core/src/main/java/com/gentics/mesh/core/verticle/node/BinaryFieldHandler.java [hunk 2]
```diff
@@ -227,57 +229,60 @@ public void handleUpdateBinaryField(InternalActionContext ac, String uuid, Strin
...
-					} catch (Exception e) {
-						log.error("Could not load schema for node {" + node.getUuid() + "}");
-						throw error(INTERNAL_SERVER_ERROR, "could not find upload file", e);
+						try {
+							return new FileInputStream(ul.uploadedFileName());
+						} catch (Exception e) {
+							log.error("Could not load schema for node {" + node.getUuid() + "}");
+							throw error(INTERNAL_SERVER_ERROR, "could not find upload file", e);
+						}
+						}).doOnSuccess(ii -> ac.put("imageInfo", ii));
```
Comment (Jotschi) [inline core/src/main/java/com/gentics/mesh/core/verticle/node/BinaryFieldHandler.java]: Document why you put the imageInfo in the context

File: core/src/main/java/com/gentics/mesh/core/verticle/node/BinaryFieldHandler.java [hunk 3]
```diff
@@ -227,57 +229,60 @@ public void handleUpdateBinaryField(InternalActionContext ac, String uuid, Strin
...
-							} catch (Exception e) {
-								log.error("Could not load schema for node {" + node.getUuid() + "}");
-								throw error(INTERNAL_SERVER_ERROR, "could not find upload file", e);
-							}
-						});
-					} catch (Exception e) {
-						log.error("Could not load schema for node {" + node.getUuid() + "}");
-						throw error(INTERNAL_SERVER_ERROR, "could not find upload file", e);
+						try {
+							return new FileInputStream(ul.uploadedFileName());
```
Comment (Jotschi) [inline core/src/main/java/com/gentics/mesh/core/verticle/node/BinaryFieldHandler.java]: Check why the stream is opened here and not closed - Why not use a try-with-resource here?

File: core/src/test/resources/logback-test.xml [hunk 4]
```diff
@@ -10,7 +10,7 @@
 			</pattern>
 		</encoder>
 		<filter class="ch.qos.logback.classic.filter.LevelFilter">
-			<level>ERROR</level>
+			<level>INFO</level>
```
Comment (Jotschi) [inline core/src/test/resources/logback-test.xml]: Revert the changes in this file.

File: core/src/main/java/com/gentics/mesh/core/verticle/node/BinaryFieldHandler.java (hunk 3 above)
Comment (philippguertler) [reply core/src/main/java/com/gentics/mesh/core/verticle/node/BinaryFieldHandler.java]: This is already done in the readImageInfo method.
